pubspec.yaml

```diff
@@ -44,4 +44,3 @@ dev_dependencies:
 
 dependency_overrides:
   scheduled_test: '^0.12.5'
-  stack_trace: {path: ../stack_trace}
```

Comment:
Yes. We're not using any new features—the tests just happen to depend on the stringification specifics.